scripts/readme.js

```diff
@@ -0,0 +1,39 @@
+const fs = require("fs");
+
+const iso = require("iso-country-codes");
+
+const CODES = fs.readdirSync("./assets");
+
+CODES.forEach(code => {
+  const name = iso.byAlpha2[code].name;
```

Comment:
Consider moving into some global file, to not repeat this in `readme.js` and `package.js`